gemfire-shared/src/main/resources/store-log4j.properties

```diff
@@ -66,15 +68,26 @@ log4j.logger.org.apache.spark.broadcast.TorrentBroadcast=WARN
 log4j.logger.org.apache.spark.ContextCleaner=WARN
 log4j.logger.org.apache.spark.MapOutputTracker=WARN
 log4j.logger.org.apache.spark.scheduler.TaskSchedulerImpl=WARN
-log4j.logger.org.apache.spark.storage.CoarseGrainedSchedulerBackend=WARN
 log4j.logger.org.apache.spark.storage.ShuffleBlockFetcherIterator=WARN
 log4j.logger.org.apache.spark.scheduler.DAGScheduler=WARN
 log4j.logger.org.apache.spark.scheduler.TaskSetManager=WARN
 log4j.logger.org.apache.spark.scheduler.FairSchedulableBuilder=WARN
 log4j.logger.org.apache.spark.scheduler.cluster.CoarseGrainedSchedulerBackend$DriverEndpoint=WARN
 log4j.logger.org.apache.spark.storage.BlockManagerInfo=WARN
-log4j.logger.org.apache.spark.executor.SnappyExecutor=WARN
+log4j.logger.org.apache.hadoop.hive=WARN
+log4j.logger.org.apache.spark.sql.catalyst.expressions.codegen.CodeGenerator=WARN
+log4j.logger.org.apache.spark.sql.execution.datasources=WARN
+log4j.logger.org.apache.spark.scheduler.SnappyTaskSchedulerImpl=WARN
+log4j.logger.org.apache.spark.MapOutputTrackerMasterEndpoint=WARN
+log4j.logger.org.apache.spark.MapOutputTrackerMaster=WARN
+log4j.logger.org.apache.spark.storage.memory.MemoryStore=WARN
+log4j.logger.org.apache.spark.MapOutputTrackerWorker=WARN
+log4j.logger.org.apache.parquet=WARN
+log4j.logger.org.apache.hadoop.io.compress=WARN
+log4j.logger.spark.jobserver.LocalContextSupervisorActor=WARN
+log4j.logger.spark.jobserver.JarManager=WARN
+log4j.logger.org.apache.spark.sql.hive.HiveClientUtil=WARN
+log4j.logger.org.datanucleus=ERROR
 
 # log4j.logger.org.apache.spark.sql.execution.WholeStageCodegenExec=DEBUG
-# log4j.logger.org.apache.spark.sql.catalyst.expressions.codegen.CodeGenerator=DEBUG
```

Comment:
This was provided if someone needs to see generated code for on-the-fly calls like in GenerateUnsafeProjection (which are not WholeStageCodegenExec). You can add comment to that effect and add it back.

Also last one is for our non-plan generated code for complex type serialization/deserialization, SparkSQLExecuteImpl etc, so can add comment like
```
# for SnappyData non-plan generated code like ComplexTypeSerializer"
```